source/libs/utils.js

```diff
@@ -40,3 +40,9 @@ export const observeEl = (el, listener, options = {childList: true}) => {
 	observer.observe(el, options);
 	return observer;
 };
+
+export const domify = html => {
+	const div = document.createElement('div');
```

Comment:
Hi, I've finished the change but I have a doubt,
here we need to use a `template` tag ?

I saw the difference between the 2 elements but with the template tag my elements are inside a fragment of a document and do not take the global style sheets

Reply:
I think what you have is fine.